example/schema.json

```diff
@@ -0,0 +1,245 @@
+{
```

Comment:
does this example adequately cover all the naming issues we're ultimately trying to fix?